### DISCUSSION TITLE
Hacker Alert trying to get video stream running

### DISCUSSION THREAD

**Discussion by DVoorhees1957** [GerryDazoo/Slinger#217] [Q&A] [answered]
I saw at least one other discussion with "Hacker Alert" in the subject, but that was a person who successfully got a video stream going and was having problems with sending remote commands. My situation is different: I can't get a video stream going. Slingbox server comes up, but when I try to stream the following URL in VLC I get a Hacker Alert message in my command prompt window every few seconds:

Network stream: http://192.168.1.124:8080/sb500

Console messages
C:\Tools\Slinger>slingbox_server.py config.ini
Version : 3.08c Running on Windows-7-6.1.7601-SP1 pid= 10824
Using config file config.ini
Connection Manager Running on port 8080 with 2 max streams using URL slingbox.
BOXES [('sb500', '500')]
Streamer Running:  2 config.ini 500 500 8080 8388608
starting up on  port 8080 8388608
Checking for slingbox at ('192.168.1.102', 5206)
('192.168.1.102', 5206) OK
sb500 Using slingbox at  ('192.168.1.102', 5206)
Streamer:  sb500 Waiting for first stream, flushing any IR requests that arrive
while not connected to slingbox
Building page for 500
 * Serving Flask app 'slingbox_server'
 * Debug mode: off
Hacker Alert. Invalid Request from  ('192.168.1.124', 62938)
GET /sb500 HTTP/1.1
Host: 192.168.1.124:8080
Accept: */*
Accept-Language: en_US
User-Agent: VLC/3.0.11 LibVLC/3.0.11
Range: bytes=0-

Hacker Alert. Invalid Request from  ('192.168.1.124', 62939)
GET /sb500 HTTP/1.0
Host: 192.168.1.124:8080
User-Agent: VLC/3.0.11 LibVLC/3.0.11
Icy-MetaData: 1

Hacker Alert. Invalid Request from  ('192.168.1.124', 62940)
GET /sb500 HTTP/1.1
Host: 192.168.1.124:8080
Accept: */*
Accept-Language: en_US
User-Agent: VLC/3.0.11 LibVLC/3.0.11
Range: bytes=0-

Hacker Alert. Invalid Request from  ('192.168.1.124', 62941)
GET /sb500 HTTP/1.0
Host: 192.168.1.124:8080
User-Agent: VLC/3.0.11 LibVLC/3.0.11
Icy-MetaData: 1

config.ini file contents
[SLINGBOXES]
sb500=500

[500]
sbtype="350/500/M1/M2"
password=<edited out>
ipaddress=192.168.1.102
port=5206
;finderid=xx
name=sb500
; Default resolution when server starts; valid range = 1-15. You can change this and reconnect without having to restart the server
Resolution=12
; FrameRate valid values: 1, 6, 10, 15, 20, 30, 60 (depending on resolution)
FrameRate=60
; VideoBandwidth valid values: 50 - 8000
VideoBandwidth=2000
; VideoSmoothness valid values: 0 (auto) - 63
VideoSmoothness=63
; Send Iframe every n seconds (max 30)
;IframeRate=5
; Audio Bit Rate valid values: 16, 20, 32 (default), 40, 48, 64, 96
;AudioBitRate=32
; VideoSource valid values: 0, 1, 2, 3 (depending on your hardware; corresponds to one of Composite, Component, HDMI or Tuner)
; ProHD: 0=Tuner, 1=composite 2=svideo 3=component
; Solo/M1: 0=composite 1=Svideo 2=component
; 500: 0=composite, 1=component, 2=HDMI
; If you don't set this the code will use the last configured input
; I recommend setting this value. If not set correctly you'll often only see a black screen 
VideoSource=1
; other variables, not all documented
;maxstreams=3
;RemoteLock=yes
;Remote=500_remote.txt

[SERVER]
; local port number for the server to listen on for connections
; Send all streaming and Remote request to this port number
port=8080
maxremotestreams=2
enableremote=yes

**Comment by GerryDazoo** [comment, marked as answer]
http://blahblah:8080/slingbox/500

slingbox is default URLbase, 500 is the name of your slingbox [500]
The name field (sb500) is just used for the server logs and is useful if you're not using the unified_config.ini. It will default to 500 if left unspecified

**Reply by DVoorhees1957** [reply]
Thanks, that fixed things! Rather than using HTTP commands to send commands to the Remote, I'm using the iPhone app that tom76017 suggested in discussion #30 to send commands directly to my TiVo. I was able to bring up the TiVo menu, change back to live TV, and change channels a few times before something caused the slingbox_server app to crash on my laptop. I wasn't able to capture the console log contents before the window closed, so I don't have much in the way of specifics - I probably triggered whatever the problem was by changing channels rapidly in succession.

Anyway, add me to the list of people that are incredibly happy with the work that you've done, to keep my Slingbox investments from going to waste!!

**Reply by GerryDazoo** [reply]
Please send me the logs. The server shouldn't crash.

**Reply by DVoorhees1957** [reply]
I don't have access to the console log any more (I started the Python code from a command prompt, and I think that log disappears when the command prompt window closes). Is there a log somewhere else that persists across slingbox_server starts and stops?

**Comment by socelillyhammer** [comment]
Im trying to redirect port to 65432 as per the release notes and the unified_config.  I also am tyring to adjust it on my router as well.  I keep getting a Hacker Alert.  What do I do to fix?


![Screenshot 2022-11-29 030104](https://user-images.githubusercontent.com/119097385/204472286-7a0e776b-aad7-4dda-aa71-fe38b27b06c9.png)

**Reply by GerryDazoo** [reply]
It looks like you should use http://ip:65432/Gerry not slingbox.
slingbox is the default but you've changed that with the [SERVER] URLbase setting.
please post your config.ini.

**Reply by socelillyhammer** [reply]
worked  thanks.

[SLINGBOX]
sbtype="350/500/M1"
password=abc123
;finderid=xyz098
ipaddress=192.168.117.110
port=5201
name=Brooklyn
Resolution=12
FrameRate=60
VideoBandwidth=2000
VideoSmoothness=63
VideoSource=2
maxstreams=3
StartChannel=602
RemoteLock=yes
Remote=500_remote.txt

;Send Iframe every n seconds Max 30
;IframeRate=5

;Audio Bit Rate. Valid Options 16, 20, 32, 40, 48, 64, 96 Default 32
;AudioBitRate=32

;Video Source  0, 1, 2, 3 depending on your hardware corresponds to one of
;Composite, Component, S-Video, HDMI or Tuner.
;Pro:        0=(Analog)Tuner   1=Composite   2=S-Video     3=Component (Via HDMI to Component Cable)
;ProHD:      0=Tuner           1=Composite   2=S-Video     3=Component
;Solo:       0=Composite       1=S-Video     2=Component
;500:        0=Composite       1=Component   2=HDMI
;350/M1/M2:  0=Composite       1=Component
; If you don't set this, the code will use the last configured input
; I recommend setting this value. If not set correctly, you'll often only see a black screen.
; Note: Setting invalid options may cause unexpected hardware behavior. For example, setting
;  2 or 3 for M2 hardware causes it to reboot.

[SERVER]
; local port number for the server to listen on for connections
; Send all streaming and Remote request to this port number.
port=65432
maxremotestreams=2
URLbase=Gerry
enableremote=yes

[REMOTE]
;see release notes for changing this if your remote doesn't work, But as a start make it 
;the same as your configured VideoSource 0-3. See above.
code=2
;path to an external remote control definitions file, if the defaults
;aren't working for you or you want to change the format of the web page.
;use the supplied remote.txt as a starting point.
;include=remote.txt
